### PR TITLE
Used styled-component to set up backbone

### PR DESCRIPTION
**Pull request by dekim2324**

### CODE REVIEW

File: client/src/components/section3/Modal.jsx [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+    `
+
+    const Email = styled.div`
+        width: 420px;
+        height: 23px;
+        display: inline-block !important;
+        position: relative !important;
+        text-align: center !important;
+        transition-property: background, border-color, color !important;
+        transition-duration: 0.2s !important;
```
Comment (minglewook) [inline client/src/components/section3/Modal.jsx]: seriously awesome styled components!

File: client/src/components/section2/Experience.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        let randomIdx = Math.floor(Math.random() * 100)
+        return(
+            <Container >
+                <Key size={13} color='#484848'/>
+                <Title>
+                {this.props.house.length > 0 && this.props.house[randomIdx].check.experience}
+                </Title>
+                <Descrip >
+                    <span>{this.props.house.length > 0 && this.props.house[randomIdx].check.experienceDescrip.slice(0, 35)
+}</span>
```
Comment (RachNovo) [inline client/src/components/section2/Experience.jsx]: You could put this between the span tags so it's on one line. @dekim2324

File: client/src/components/section2/Location.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+import styled from 'styled-components';
+import { MapPin } from 'react-feather';
+
+class Location extends Component {
+    constructor(props) {
+        super(props)
+    }
+
+    render() {
+        let randomIdx = Math.floor(Math.random() * 100)
```
Comment (RachNovo) [inline client/src/components/section2/Location.jsx]: I think this name could cause errors if you're rewriting it. Might just be my preference. @dekim2324

File: client/src/components/App.test.js [hunk 4]
```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import { shallow, mount } from 'enzyme';
+
+import App from './App.jsx';
+import Section1 from './section1/Section1.jsx';
+
+// describe('<App />', () => {
```
Comment (RachNovo) [inline client/src/components/App.test.js]: Could you get rid of the commented out code? @dekim2324

File: client/src/components/section3/Section3.jsx [hunk 5]
```diff
@@ -1,17 +1,128 @@
...
     constructor(props) {
-        super(props)
-    }
+        super(props);
+
+
+        this.state = {
+            status: 'collapse',
+            show: false,
+            idx:  Math.floor(Math.random() * 100)
```
Comment (RachNovo) [inline client/src/components/section3/Section3.jsx]: Is there a reason you put this here instead of as a variable in the render function? @dekim2324

**Comment by RachNovo** [comment]
Looks great Dennis! I haven't done styling yet so I didn't comment on your styled components. Good job :)

File: client/src/components/section1/Section1.jsx [hunk 6]
```diff
@@ -1,17 +1,113 @@
...
             </div>
         )
     }
 }
 
+const Line = styled.div`
+    border-bottom-width: var(--border-rule-border-width, 1px) !important;
+    border-bottom-color: var(--color-divider, #EBEBEB) !important;
+    border-bottom-style: solid !important;
+`;
```
Comment (therobinkim) [inline client/src/components/section1/Section1.jsx]: Why do these need `!important`?

And where are these CSS variables being used?

File: client/src/components/section3/Modal.jsx [hunk 7]
```diff
@@ -0,0 +1,228 @@
...
+        min-width: 71.1935px !important;
+        box-shadow: none !important;
+        color: rgb(255, 255, 255) !important;
+        text-decoration: none !important;
+        border-radius: 4px !important;
+        border-width: 2px !important;
+        border-style: solid !important;
+        background: rgb(255, 90, 95) !important;
+        border-color: transparent !important;
+        margin: 20px auto 0 auto;
```
Comment (therobinkim) [inline client/src/components/section3/Modal.jsx]: I do not recommend copy/pasting CSS for this project.

File: package.json [hunk 8]
```diff
@@ -11,27 +11,44 @@
...
     "babel-loader": "^8.0.6",
+    "babel-polyfill": "^6.26.0",
     "body-parser": "^1.19.0",
+    "eslint": "^6.5.1",
     "express": "^4.17.1",
     "faker": "^4.1.0",
+    "jest-cli": "^24.9.0",
+    "jest-fetch-mock": "^2.1.2",
+    "mongo": "^0.1.0",
+    "mongod": "^2.0.0",
```
Comment (therobinkim) [inline package.json]: What are these packages?

File: package.json [hunk 9]
```diff
@@ -11,27 +11,44 @@
...
     "mongodb": "^3.3.2",
     "mongoose": "^5.7.4",
+    "morgan": "^1.9.1",
     "react": "^16.10.2",
     "react-dom": "^16.10.2",
+    "react-feather": "^2.0.3",
     "react-native": "^0.61.2",
```
Comment (therobinkim) [inline package.json]: What are we using react-native for?

**Pull request merged**